pyproject.toml

```diff
@@ -29,8 +29,8 @@ dependencies = [
 ]
 
 [tool.black]
-line-length = 100
-required-version = "22.10.0" # Black will refuse to run if it's not this version.
+line-length = 88
```

Comment:
This is the flake8 configured line length

Reply:
Sneaking these changes in since they are just ide settings but happy to decouple